test_runner.py

```diff
@@ -273,6 +273,19 @@ def build_test_list():
             "fsdp2_mem_tracker",
             ngpu=4,
         ),
+        OverrideDefinitions(
+            [
+                [
+                    "--training.tensor_parallel_degree 1",
+                    "--training.data_parallel_degree 8",
+                    "--experimental.data_parallel_type ddp",
+                    "--experimental.enable_compiled_autograd",
+                ]
+            ],
+            "CompiledDDP",
+            "compiled_ddp",
+            ngpu=8,
```

Comment:
why do we need to use 8 gpus? A side note is that it doesn't mean it passes the 4 gpu test.